packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -23,6 +24,7 @@ export interface IComponentWrapperProps<T> {
     singleRow?: boolean;
     designSystem: T;
     view: ComponentViewTypes;
+    dir: Direction;
```

Comment:
Looks like the comment disappeared here, so bringing it back up. 

I think this should be optional.